lib/app_web/controllers/page_controller.ex

```diff
@@ -12,25 +12,24 @@ defmodule AppWeb.PageController do
   def index(conn, _params) do
     url_oauth_google = ElixirAuthGoogle.generate_oauth_url(conn)
     changeset = Person.changeset_registration(%Person{}, %{})
-    render(conn, "index.html",
-      [url_oauth_google: url_oauth_google, changeset: changeset])
+    render(conn, "index.html", url_oauth_google: url_oauth_google, changeset: changeset)
```

Comment:
After running `mix format` this line is now 88 characters long ... 
can we add `line_length: 80` to `.formatter.exs` to avoid super-long-lines?
see: https://medium.com/blackode/code-formatter-the-big-feature-in-elixir-v1-6-0-f6572061a4ba

Reply:
Added  `line_length: 80` :+1: